### ISSUE TITLE
Add More Recording Functions

### ISSUE THREAD

**Issue by zo-shin** [segler-alex/RadioDroid#476]
Maybe some of the following functions are useful:
1 - auto-record on playing
2 - scheduled recording
3 - record current track only
4 - split recording file by track name

**Comment by werman** [comment]
> Maybe some of the following functions are useful:
> 1 - auto-record on playing

What would be use case the this feature?

> 2 - scheduled recording

What would be use case the this feature?

> 3 - record current track only

Probably would be useful

> 4 - split recording file by track name

Yes, that would be good and was suggested several times.

**Comment by zo-shin** [comment]
If users often record a certain program at a fixed time, 1 or 2 will be useful. Maybe it's rare.
For 3 & 4, they already exist as paid functions in another app.

**Comment by Willkha** [comment]
> Maybe some of the following functions are useful:
> 1 - auto-record on playing
> 2 - scheduled recording
> 3 - record current track only
> 4 - split recording file by track name

I've been looking for this feature.
Especially the recording schedule in the week.
Many people cannot listen to their favorite programs because they are working or studying.

**Comment by segler-alex** [comment]
5. scheduled stop of a recording

**Comment by exquo** [comment]
> 4 - split recording file by track name

FR issue: #276

**Comment by MainTomato** [comment]
I agree that scheduled recordings can be really useful. Some of the best Radio shows happen during the night so it can be awkward to catch up on them. It would be better then having to download something like BBC Sounds.
